packages/opentelemetry-exporter-stackdriver-trace/README.md

```diff
@@ -0,0 +1,98 @@
+# OpenTelemetry Stackdriver Trace Exporter
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+OpenTelemetry Stackdriver Trace Exporter allows the user to send collected traces to Stackdriver.
+
+[Stackdriver Trace](https://cloud.google.com/trace) is a distributed tracing system. It helps gather timing data needed to troubleshoot latency problems in microservice architectures. It manages both the collection and lookup of this data.
+
+## Setup
+
+Stackdriver Trace is a managed service provided by Google Cloud Platform.
+
+### Installation
+
+Install the npm package `@opentelemetry/exporter-stackdriver-trace`
+
+```shell
+$ npm install @opentelemetry/exporter-stackdriver-trace
+```
+
+### Authentication
+
+The Stackdriver Trace exporter supports authentication using service accounts. These can either be defined in a keyfile (usually called `service_account_key.json` or similar), or by the environment. If your application runs in a GCP environment, such as Compute Engine, you don't need to provide any application credentials. The client library will find the credentials by itself. For more information, go to <https://cloud.google.com/docs/authentication/>.
+
+#### Service account key
+
+If you are not running in a GCP environment, you will need to give the service account credentials to the exporter.
+
+```js
+const { StackdriverTraceExporter } = require('@opentelemetry/exporter-stackdriver-trace');
+
+const exporter = new StackdriverTraceExporter({
+  serviceName: 'your-application-name',
```

Comment:
Just curious to know the origin of this, OC never had a `serviceName` concept for Stackdriver exporter.

Reply:
It was in their api documentation so I added it.

Reply:
I see, could you please share the link (for the reference)?

Reply:
actually it seems I was wrong. I honestly don't know where it came from... I'm removing it.

Reply:
Could you please remove `serviceName` from README file.